product_replenishment_cost/models/product_product.py

```diff
@@ -28,14 +12,15 @@
 class ProductProduct(Model):
     _inherit = 'product.product'
 
-    @api.one
+    @api.multi
     @api.depends('product_tmpl_id.standard_price')
     def _get_replenishment_cost(self):
-        self.replenishment_cost = self.standard_price
+        for product in self:
+            product.replenishment_cost = product.standard_price
 
     replenishment_cost = fields.Float(
-        compute=_get_replenishment_cost, store=True,
```

Comment:
Ok, I see, and I don't find a trigger for that. With the explanation, it's fine.

If the issue is with the other module, maybe you can leave `store=True` here, and explicitely change to `store=False` in the other module ? (with a comment explaining why)